kong/db/schema/init.lua

```diff
@@ -930,6 +930,7 @@ function Schema:validate_field(field, value)
     local field_schema = get_field_schema(field)
     -- TODO return nested table or string?
     local copy = field_schema:process_auto_fields(value, "insert")
+    -- Changes in custom entity check only applies to copy, not to value
```

Comment:
Not sure if this is intended or bug. Leave a comment for reminder.

Reply:
I think as this is `validate_field` function. The process auto fields is just executed so that the validation can pass or something. Thus I don't think we need to add this comment. Or if we add the comment, the comment should explain why copy, and not to point something that might be totally false. Thus I lean removing this comment as bad comment is worse than no comment at all.

```suggestion
```

Reply:
Agree with:

```
TODO: explain why we need to make a copy?"
```

The issue is `field_schema:validate(copy)` may have config customizations. If so, the customizations only go to the `copy`, not the original `value`.

Reply:
@outsinre, yes, the customizations I think are fine. This is validation time, the validation should not change the entitity I think. So that's why it is probably copied so that it does not change the original entity on validation. Perhaps it does the change on some other time.